test_conformance/api/test_kernels.cpp

```diff
@@ -370,87 +388,6 @@ int test_set_kernel_arg_by_index(cl_device_id deviceID, cl_context context, cl_c
     return 0;
 }
 
-int test_set_kernel_arg_struct(cl_device_id deviceID, cl_context context, cl_command_queue queue, int num_elements)
```

Comment:
We do not need this test ?

Reply:
This test was passing a structure containing `cl_mem` objects as a kernel argument, which is not valid. I assume there is a (valid) test that covers this in the SVM suite. The test was not being used, presumably because it is not valid and would not pass on many implementations.

Reply:
This is worth double-checking (the SVM coverage). Also, should we be removing the program source as well? I'm tempted to suggest this should have been a separate PR.

Reply:
[`test_set_kernel_exec_info_svm_ptrs.cpp`](https://github.com/KhronosGroup/OpenCL-CTS/blob/master/test_conformance/SVM/test_set_kernel_exec_info_svm_ptrs.cpp) does this the right way.

I can submit the removal of this test (and the source - good catch!) as a separate PR. Just need to fix CI for OpenCL 3.0 first ;-)